components/symbol/src/Symbol.js

```diff
@@ -49,8 +50,16 @@ export const Symbol = ({
 	children,
 	...props
 }) => {
+	const { [pkg.name]: overridesWithTokens } = useBrand();
+
+	const overrides = {
+		Wrapper: SymbolWrapper,
+		svgAttributes: {},
```

Comment:
We also need to spread css for each component. I forgot that in `Icon` as well.

```suggestion
		svgAttributes: {},
		css {},
```